src/QMCDrivers/WFOpt/HamiltonianRef.cpp

```diff
@@ -59,14 +59,14 @@ int HamiltonianRef::addObservables(ParticleSet& P)
   P.Collectables.rewind();
   for (int i = 0; i < Hrefs_.size(); ++i)
     Hrefs_[i].get().addObservables(Observables, P.Collectables);
-  int myIndex = P.PropertyList.add(Observables.Names[0]);
+  int my_index_ = P.PropertyList.add(Observables.Names[0]);
```

Comment:
function local variable should not have underscore. Also add const if there is no intention to modify.